dasharo-compatibility/306-usb-hid-and-msc-support.md: update description tests USB001.003 and USB002.003 (#172)
Dasharo/docs: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md

1. Open PowerShell and and run the follwing command:

    ```bash
    Get-PnpDevice -PresentOnly | Where-Object { $_.InstanceId -match '^USB' }
    ```

> 1. Note results.

Reviewer: ```suggestion
1. Note the results.
```

**Expected result**

> Example output:

Reviewer: Indentation missing

    ```bash
    Status     Class           FriendlyName
    ------     -----           ------------
    OK         DiskDrive       Mass Storage Device USB Device
    OK         USB             Generic USB Hub
    OK         HIDClass        USB Input Device
    OK         Bluetooth       Intel(R) Wireless Bluetooth(R)
    OK         USB             USB Root Hub (USB 3.0)
    OK         Net             TP-LINK Gigabit Ethernet USB Adapter
    OK         USB             Generic USB Hub
    OK         USB             USB Mass Storage Device
    ```

> Connect additional USB device, run command once again and note reults.

Reviewer: ```suggestion
Connect additional USB device, run command once again and note the results.
```

> Connect additional USB device, run command once again and note reults.

Reviewer: indentation missing

1. Power on the DUT.
1. Boot into the system.
1. Log into the system by using the proper login and password.
1. Open PowerShell and and run the follwing command:

    ```bash
    Get-CimInstance win32_KEYBOARD
    ```

1. Note results.

**Expected result**

> Informations about connected keyboard should be displayed.

Reviewer: ```suggestion
1. Information about connected keyboard should be displayed.
```

Author: add all improvements

> Informations about connected keyboard should be displayed.

Reviewer: indentation missing
